glossary/pier.md

```diff
@@ -0,0 +1,18 @@
++++
+title = "Pier"
+template = "doc.html"
+[extra]
+category = "arvo"
++++
+
+A **pier** is the directory which contains the state of an urbit [ship](../ship). It is automatically created in the current directory when booting a ship for the first time.
+
+Your pier should be kept safe - if it's deleted, you will need to perform a [breach](../breach) in order to advise the rest of the network that your ship's state has been lost.
+
+Note that a backup should usually _not_ be taken of your pier as a means of keeping it safe - once the ship is run on the network, the backup will be outdated and running from it will necessitate a breach. A running ship automatically backs up its own state within the pier directory for recovery purposes, so as long as the pier is preserved, recovery from most problems is possible.
+
+### Further Reading
```

Comment:
We try to have bidirectional links in the glossary between closely related topics. I would suggest linking `ship`, `breach` and `eventlog` to this page as well.